pkg/ebpf/perf.go

```diff
@@ -68,6 +70,7 @@ func (c *PerfHandler) RecordHandler(record *perf.Record, perfMap *manager.PerfMa
 		return
 	}
 
+	ReportPerfMetrics(perfMap.Name, ebpf.PerfEventArray.String(), record.CPU, record.Remaining, perfMap.BufferSize())
```

Comment:
Any idea of the overhead of doing `prometheus.Set` directly in the callback? Maybe comparing lost event count before and after this would be useful.

Reply:
Looks like it takes a single read mutex lock in the common case, but we can do some more performance testing before merging this.